archey/logos/endeavouros.py

```diff
@@ -0,0 +1,27 @@
+# Endeavour OS Logo
+
+from archey.colors import Colors
+
+COLORS = [Colors.RED_BRIGHT, Colors.MAGENTA_NORMAL, Colors.BLUE_NORMAL]
+
+LOGO = [
+    """                                        """,
+    """                                        """,
+    f"""{COLORS[0]}                     ./{COLORS[1]}o{COLORS[2]}.               """,
+    f"""{COLORS[0]}                   ./{COLORS[1]}sssso{COLORS[2]}-             """,
+    f"""{COLORS[0]}                 `:{COLORS[1]}osssssss+{COLORS[2]}-           """,
+    f"""{COLORS[0]}               `:+{COLORS[1]}sssssssssso{COLORS[2]}/.         """,
+    f"""{COLORS[0]}             `-/{COLORS[1]}ssssssssssssso{COLORS[2]}/.        """,
+    f"""{COLORS[0]}           `-/+{COLORS[1]}sssssssssssssssso{COLORS[2]}+:`     """,
+    f"""{COLORS[0]}         `-:/+{COLORS[1]}sssssssssssssssssso{COLORS[2]}/.     """,
+    f"""{COLORS[0]}       `.://o{COLORS[1]}sssssssssssssssssssso{COLORS[2]}++-   """,
+    f"""{COLORS[0]}      .://+{COLORS[1]}ssssssssssssssssssssssso{COLORS[2]}++:  """,
+    f"""{COLORS[0]}    .:///o{COLORS[1]}ssssssssssssssssssssssssso{COLORS[2]}++: """,
+    f"""{COLORS[0]}  `:////{COLORS[1]}ssssssssssssssssssssssssssso{COLORS[2]}+++.""",
+    f"""{COLORS[0]}`-////+{COLORS[1]}ssssssssssssssssssssssssssso{COLORS[2]}++++-""",
+    f"""{COLORS[0]} `..-+{COLORS[1]}oosssssssssssssssssssssssso{COLORS[2]}+++++/`""",
+    f"""{COLORS[2]}   ./++++++++++++++++++++++++++++++/:.  """,
+    f"""{COLORS[2]}  `:::::::::::::::::::::::::------``    """
+    ]
+
+
```

Comment:
With the changes, it looks just as intended.  I have removed my IP addresses from the Picture.
![Bildschirmfoto vom 2024-01-05 13-09-56](https://github.com/HorlogeSkynet/archey4/assets/80016610/3e59b3e7-80e0-4fbd-9067-31903ee659a7)

> This is probably because you have "honor_ansi_color": true set in config.json. Setting it to false will use the colours from the file. Your logo currently shows up with the full set of colours as they're hardcoded into it. 

I would suggest changing the default setting to display the colors immediately. I don't know of any terminal that can't display colors. Even xterm somehow manages it. In TTY everything is fine too.

> Do you know whether Archey detects EndeavourOS properly? E.g. does your logo show up if you directly run Archey as a module (python3 -m archey), and what shows up in the Distro line? We may have to write some custom logic to detect EndeavourOS, as I'm not sure our current solution will find it.

I didn't want to mess up my system, so I have a copy in my home folder. There is a startup file there that calls archey. It is exactly the same as the one in /bin/. python3 -m archey will fail.


start.py  (location: [root Folder])
```python
#!/usr/bin/python
# -*- coding: utf-8 -*-
import re
import sys
from archey.__main__ import main
if __name__ == "__main__":
    sys.argv[0] = re.sub(r"(-script\.pyw|\.exe)?$", "", sys.argv[0])
    sys.exit(main())

```

Reply:
> I would suggest changing the default setting to display the colors immediately. I don't know of any terminal that can't display colors. Even xterm somehow manages it. In TTY everything is fine too.

Ah sorry, I think you misunderstood what I meant -- the option uses `ANSI_COLOR` as defined in `/etc/os-release` instead of the colours specified in the file -- I agree it should probably be off by default. I believe we do respect some environment variables that request disabling colour regardless.

> python3 -m archey will fail.

Interesting, it should pick the current working directory module and import that rather than the system module. Well, as long you can get it working, it's fine either way.